core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -201,31 +204,43 @@ public RestStatus status() {
     }
 
     /**
-     * Gets the location of the written document as a string suitable for a {@code Location} header.
-     * @param routing any routing used in the request. If null the location doesn't include routing information.
+     * Return the relative URI for the location of the document suitable for use in the {@code Location} header. The use of relative URIs is
```

Comment:
I think it's odd that we would have a 140 character limit on Java source code but only 72 characters on Javadocs (and comments?). If you feel strongly about this, I think that we should discuss it more broadly.